pages/404.tsx

```diff
@@ -2,54 +2,56 @@ import Link from 'next/link'
 
 const Custom404 = () => {
   return (
-    <div className="grid md:grid-cols-2 sm:grid-cols-1 items-center justify-center overflow-visible md:h-96 sm:h-screen mx-2 my-2 px-20">
-      <div className="error-404">
-        <h1 className="text-2xl">{"We couldn't find that Web page"}</h1>
-        <h2>An error 404 occured on server</h2>
-        <p>
-          {
-            "We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for. What next?"
-          }
-        </p>
-        <ul>
-          <li>
-            Return to the{' '}
-            <Link href="/" locale="default">
-              <a className="text-cyan-600 underline">home page</a>
-            </Link>
-            ;
-          </li>
-          <li>
-            <Link href="https://www.canada.ca/en/contact.html">
-              <a className="text-cyan-600 underline">Contact us</a>
-            </Link>
-            {" and we'll help you out."}
-          </li>
-        </ul>
-      </div>
-      <div className="error-404">
-        <h1 className="text-2xl">Nous ne pouvons trouver cette page Web</h1>
-        <h2>Erreur 404</h2>
-        <p>
-          {
-            "Nous sommes désolés que vous ayez abouti ici. Il arrive parfois qu'une page ait été déplacée ou supprimée. Heureusement, nous pouvons vous aider à trouver ce que vous cherchez. Que faire?"
-          }
-        </p>
-        <ul>
-          <li>
-            Retournez à la{' '}
-            <Link href="/" locale="default">
-              <a className="text-cyan-600 underline">page {"d'accueil;"}</a>
-            </Link>
-          </li>
-          <li>
-            <Link href="https://www.canada.ca/en/contact.html">
-              <a className="text-cyan-600 underline">Communiquez avec nous</a>
-            </Link>
-            {" pour obtenir de l'aide."}
-          </li>
-        </ul>
-      </div>
+    <div className="min-h-screen">
```

Comment:
I've played with tailwind classes and I think this would looks good;

```html
<div className="container mx-auto">
      <section className="grid grid-cols-1 lg:grid-cols-2 mx-4 lg:mx-6 gap-4 lg:gap-8">
        <div>English content</div>
        <div>English content</div>
      </section>
</div>
```

Reply:
Gap utility classes help to space grid or flexbox items.
https://tailwindcss.com/docs/gap

Reply:
Sure, I like how this looks as well and it's less confusing. I originally played with `gap`, but ended up taking a different approach and going overboard I guess 😄